opflexagent/utils/ep_managers/endpoint_file_manager.py

```diff
@@ -226,6 +240,14 @@ def _setup_ep_directory(self):
                     filename = f[:-len(FILE_EXTENSION) - 1]
                     if '_' in f:
                         self._registered_endpoints.add(f.split('_')[0])
+                        fp = file(os.path.join(directory, f))
+                        ep_opts = json.load(fp)
```

Comment:
you need to gracefully handle json parsing errors here?

Reply:
ok